### PR TITLE
Replace Markdown links in HTML with HTML links

### PR DESCRIPTION
**Pull request by haltman-at**
This PR replaces the Markdown links inside HTML on the debugger documentation page with HTML links, so that they'll work.

### CODE REVIEW

File: src/docs/truffle/overview.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ layout: docs.hbs
 <img style="max-width: 160px;" src="/img/truffle-logo-dark.svg" alt="Truffle Logo" />
 
 <div class="text-center docs-badges">
-  [![Join the chat at https://gitter.im/consensys/truffle](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/consensys/truffle?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+  <a href="https://gitter.im/consensys/truffle?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge">
```
Comment (seesemichaelj) [inline src/docs/truffle/overview.md:8]: can you actually fix this in the other `overview.md` files while you're at it?

**Comment by haltman-at** [comment]
This is actually not done yet!  I put this up before realizing the extent of the problem.  Will keep finding more and re-review.

**Comment by haltman-at** [comment]
> What grep command did you use to find these (while not finding all markdown links)?

It was mostly manual, TBH...

File: src/docs/truffle/overview.md (hunk 1 above)
Comment (haltman-at) [reply src/docs/truffle/overview.md:8]: Done!

**Pull request merged**